docs/LearningOutcomes.adoc

```diff
@@ -38,7 +38,7 @@ Note how the <<DeveloperGuide#architecture, Developer Guide>> uses events to com
 
 == Use API Design `[LO-ApiDesign]`
 
-Note how components of AddressBook have well-defined APIs. For example, the API of the `Logic` component is given in the link:{repoURL}/src/main/java/seedu/location/logic/Logic.java[`Logic.java`]
+Note how components of AddressBook have well-defined APIs. For example, the API of the `Logic` component is given in the link:{repoURL}/src/main/java/seedu/venue/logic/Logic.java[`Logic.java`]
```

Comment:
I think we should skip renaming these
I think it was originally address, then became location